tests/suites/test_suite_psa_crypto.function

```diff
@@ -491,6 +491,10 @@ static int exercise_aead_key( mbedtls_svc_key_id_t key,
     size_t ciphertext_length = sizeof( ciphertext );
     size_t plaintext_length = sizeof( ciphertext );
 
+    /* Default IV length for AES-GCM is 12 bytes */
+    if( (alg & ~PSA_ALG_AEAD_TAG_LENGTH_MASK) == PSA_ALG_GCM )
```

Comment:
I couldn't find any. Is there one?

Reply:
See the implementation of `psa_aead_setup`:
```
if( PSA_ALG_AEAD_WITH_TAG_LENGTH( alg, 0 ) == PSA_ALG_AEAD_WITH_TAG_LENGTH( PSA_ALG_GCM, 0 ) )
```
or (a little more readable but less efficient because it has to do case analysis on the algorithm):
```
if( PSA_ALG_AEAD_WITH_DEFAULT_TAG_LENGTH( alg ) == PSA_ALG_AEAD_WITH_DEFAULT_TAG_LENGTH( PSA_ALG_GCM ) )
```
I admit it doesn't exactly roll off the tongue. Should we add `PSA_ALG_IS_GCM`? Or a more general `PSA_ALG_IS_SAME_AEAD`? Mind you, when you have a switch rather than an if, the existing macros are about as good as it gets.

Reply:
Thanks for the suggestion. Updated.